service/src/main/java/org/cbioportal/service/impl/ExpressionEnrichmentServiceImpl.java

```diff
@@ -137,9 +137,16 @@
                 .collect(Collectors.groupingBy(ReferenceGenomeGene::getEntrezGeneId));
 
         expressionEnrichments.forEach(expressionEnrichment -> {
-            ReferenceGenomeGene gene = geneMapByEntrezId.get(expressionEnrichment.getEntrezGeneId()).get(0);
-            expressionEnrichment.setHugoGeneSymbol(gene.getHugoGeneSymbol());
-            expressionEnrichment.setCytoband(gene.getCytoband());
+            try {
```

Comment:
Should we remove this(setting Hugo gene symbol and cytoband), since we are getting gene info for all genes in the front-end.

Reply:
let me check if we back fill cytobands for expression in frontend.

Reply:
you are right. We can remove set up for cytobands (it is done in, but we still need to set hugoSymbols.

Reply:
ok. I think there are couple of ways to do it
1. we could you `hugoSymbols` from the response of `reference-genome-genes` api
2. set `hugoSymbols` similar to what's done for mutation or copy number enrichments api

Reply:
@kalletlak , we have to set hugoSymbols for all alterations in the backend, not in frontend. Please check the function makeEnrichmentDataPromise see if you agree. 
```
                // add cytoband from reference gene
                for (const d of data) {
                    const refGene = params.referenceGenesPromise.result![d.hugoGeneSymbol];

                    if (refGene)
                        d.cytoband = refGene.cytoband;
                }
```

Reply:
@kalletlak , changes made. please have your final review. thanks!